pyfecons/exceptions.py

```diff
@@ -0,0 +1,9 @@
+import pyfecons
+
+class TemplateException(Exception):
```

Comment:
What's the purpose of this class? Do you intend to add more tests here?

Reply:
Yes, it's just a placeholder to mark where new exceptions should be defined; once we have implemented a new exception this line can be removed.